caravel/views.py

```diff
@@ -1793,6 +1793,12 @@ def dashboard(self, dashboard_id):
         def dashboard(**kwargs):  # noqa
             pass
         dashboard(dashboard_id=dash.id)
+        if request.args.get("standalone") == "true":
+            return self.render_template(
+                "caravel/dashboard_standalone.html",
```

Comment:
i think keeping one template for the dashboard and adding the conditions where needed will be more clear in the long run since we won't have to maintain 2 templates. the conditional for show/hiding the header should work the same way in `basic.html` for both the slice and dashboard view, so we will probably only need the conditionals in the dashboard template.